CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 ### Added
 
 - Go [type aliases](https://go.dev/ref/spec#Alias_declarations) are now generated for each WIT type alias (`type foo = bar`). Deep chains of type aliases (`type b = a; type c = b;`) are fully supported. Generated documentation now reflects whether a type is an alias. Fixes [#204](https://github.com/bytecodealliance/wasm-tools-go/issues/204).
+- `wit-bindgen-go generate` now generates a WIT file for each WIT world in its corresponding Go package directory. For example the `wasi:http/proxy` world would generate `wasi/http/proxy/proxy.wit`.
```

Comment:
thoughts on renaming the wit file to `proxy@0.2.2.wit`?

Reply:
⚖️ 
I think keeping `<world>.wit` is ok  cause we'd never have `proxy@0.2.2.wit` & `proxy@0.2.0.wit` in the same directory, but am not strong about it.
What's the motivation @Mossaka ?

Reply:
The Go package directory can be versioned, e.g. `wasi/http@0.2.2/proxy/proxy.wit`